packages/libs/coreui/src/components/Mesa/Ui/SelectionCell.jsx

```diff
@@ -43,9 +47,29 @@ class SelectionCell extends React.PureComponent {
     };
 
     return (
-      <th className="SelectionCell" onClick={handler}>
+      <th
+        className={
+          'SelectionCell' + (selectColumnHeadingDetails ? '__withDetails' : '')
+        }
+        onClick={handler}
+      >
         {inert ? null : (
-          <Checkbox checked={checked} indeterminate={isIndeterminate} />
+          <AnchoredTooltip
```

Comment:
I'm not sure we really want/need this tooltip, but I added it to hedge against complaints re: losing the "Check All" and "Clear All" buttons. To be clear, this would be rendering a tooltip in all Mesa instances with row selections.